vite.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'vite'
+
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  base: process.env.VITE_BASE || '/',
+  build: {
+    target: 'esnext' //browsers can handle the latest ES features
```

Comment:
recommend es2022, esnext is a bit too aggressive (even 2022 is aggressive tbh)

Reply:
Can you explain a bit more the difference and why

Reply:
This refers to the ECMAScript standard. ECMAScript is the official name of JavaScript, due to some copyright issue with Oracle. 

ES{Year} refers to the year of the standard that the code will be built for. Each year, new features are added to the standard. If you set an older standard, most new features are still available, but code will be injected to make sure they work in older browsers.

ESNext means the latest version of ECMAScript. Once a feature is introduced into the standard, it can take a while for browsers to implement it, and even more time for users to actually update their browser, so it is better to be a few years behind (I believe the default for Vite is ES2020).

Reply:
For more details, see https://ecma-international.org/publications-and-standards/standards/ecma-262/

Reply:
changed, thanks for the info